src/aether-producer/values.yaml

```diff
@@ -30,6 +30,7 @@ app:
   kernel:
     secret: kernel-secrets
     user: kernel
+    accessType: api
```

Comment:
you can activate `api` mode if you provide `AETHER_KERNEL_URL` and `AETHER_KERNEL_TOKEN`... is this the case?

Reply:
it is yeah

Reply:
I'll remove it then @obdulia-losantos